src/ArborX_LinearBVH.hpp

```diff
@@ -36,6 +36,42 @@ namespace Details
 struct HappyTreeFriends;
 } // namespace Details
 
+namespace Experimental
+{
+struct Morton32
```

Comment:
Not sure if this should live in this file. Maybe move it to the details or experimental Morton file.